src/config/delegates.json

```diff
@@ -1,4 +1,16 @@
-{
+i{
+ "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9": {
+    "name": "ReFiCol4🌎",
+    "address": "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9",
+    "logoUri": "/logos/validators/ReFiCol.jpg",
```

Comment:
You have to add the image or this won't work. Please use the same naming convention as the others (`refi-col.jpg`)

Reply:
Image Added